install/langs.php

```diff
@@ -48,4 +48,4 @@
   'tr_TR' => 'Turkish',
   'uk_UA' => 'Ukrainian',
   'vi_VN' => 'Vietnamese',
-);
\ No newline at end of file
+);
```

Comment:
this might come back but it's hard to remove too since I have to beat up my editor to not do this - I think we can ignore this accidental extra fix